cmd/cli/kubectl-kyverno/commands/apply/command.go

```diff
@@ -121,6 +122,7 @@ func Command() *cobra.Command {
 	cmd.Flags().BoolVarP(&table, "table", "t", false, "Show results in table format")
 	cmd.Flags().StringSliceVarP(&applyCommandConfig.Exception, "exception", "e", nil, "Policy exception to be considered when evaluating policies against resources")
 	cmd.Flags().StringSliceVarP(&applyCommandConfig.Exception, "exceptions", "", nil, "Policy exception to be considered when evaluating policies against resources")
+	cmd.Flags().BoolVarP(&applyCommandConfig.inlineExceptions, "evaluate-inline-exceptions", "", false, "Evaluate policy exception to be considered from the resources to be processed")
```

Comment:
Not sure if we really need a flag for this enhancement.

Reply:
I'm fine with that, if it would be a default feature.

Reply:
Agreed, may be best to process by default.